wweval/R/add_horizons.R

```diff
@@ -84,3 +85,22 @@ order_horizons <- function(df) {
 
   return(df_w_order)
 }
+#' Get the order of the time periods for plotting
+#'
+#' @param df a dataframe containing the column name `period` which will
+#' contain character strings indicating what period the scores correspond to
+#'
+#' @return a dataframe containing the same columns as `df` but now ordered
+#' in terms of the specified order.
```

Comment:
```suggestion
#' @return a dataframe containing the same columns as `df` but with the `period`
#' factor now ordered so that `"Oct 2023-Mar 2024"` comes first and 
#' `"Feb 2024-Mar 2024"` comes second.

```